tests/app/main/views/test_suppliers.py

```diff
@@ -1484,6 +1484,25 @@ def test_permissions(self, role, expected_code):
         response = self.client.get("/admin/suppliers/1234/draft-services")
         assert response.status_code == expected_code
 
+    @pytest.mark.parametrize("role, visible_open_frameworks", [
+        ("admin-framework-manager", 1),
+        ("admin-ccs-sourcing", 0),
+    ])
+    def test_open_frameworks_only_visible_to_framework_manager_admins(self, role, visible_open_frameworks):
+        self.data_api_client.find_frameworks.return_value = {
+            "frameworks": [
+                FrameworkStub(slug="g-cloud-8", status="open").response(),
+                FrameworkStub(slug="digital-outcomes-and-specialists-2", status="pending").response(),
+            ],
+        }
+        self.user_role = role
+
+        response = self.client.get("/admin/suppliers/1234/draft-services")
+
+        document = html.fromstring(response.get_data(as_text=True))
+        assert len(document.xpath("//h2[normalize-space(string())=$t]", t="G-Cloud 8")) == visible_open_frameworks
+        assert len(document.xpath("//h2[normalize-space(string())=$t]", t="Digital Outcomes and Specialists 2")) == 1
+
```

Comment:
Can you also add a test for at least one status != other, to check both roles can still view the frameworks?

Reply:
I've added another parametrization for the 'post-open' framework statuses (pending, standstill, live, expired) that the Sourcing admins will be able to see.